tests/Unit/InMemoryFileFetcherTest.php

```diff
@@ -4,9 +4,11 @@
 
 namespace FileFetcher\Tests\Unit;
 
+use InvalidArgumentException;
 use FileFetcher\FileFetchingException;
 use FileFetcher\InMemoryFileFetcher;
 use PHPUnit\Framework\TestCase;
+use Nette\InvalidArgumentException as NetteInvalidArgumentException;
```

Comment:
??

Reply:
Hi @JeroenDeDauw, that's my fault.

This exception class namespace is useless.

Reply:
@JeroenDeDauw. That's right.